index.html

```diff
@@ -313,7 +313,7 @@ <h2><code>PointerEvent</code> Interface</h2>
                 <dl data-dfn-for="PointerEvent" data-link-for="PointerEvent">
                     <dt><dfn>pointerId</dfn></dt>
                         <dd>
-                            <p>A unique identifier for the pointer causing the event. This identifier MUST be unique from all other <a data-lt="active pointer">active pointers</a> in the <a>top-level browsing context</a> (as defined by [[HTML]]) at the time. A user agent MAY recycle previously retired values for <code>pointerId</code> from previous active pointers, if necessary.</p>
+                            <p>A unique identifier for the pointer causing the event. This identifier MUST be a positive number and unique from all other <a data-lt="active pointer">active pointers</a> in the <a>top-level browsing context</a> (as defined by [[HTML]]) at the time. A user agent MAY recycle previously retired values for <code>pointerId</code> from previous active pointers, if necessary.</p>
```

Comment:
This is not backwards compatible.

Reply:
I suggest ruling out only "zero" as a special case (and keep allowing negative ids implicitly):

> This identifier MUST be unique from all other active pointers in the top-level browsing context at the time, **except for the value zero which indicates the event is from a non-pointing device**.

Reply:
took a slightly different approach to wording ... see if that makes sense though